docs/development.rst

```diff
@@ -6,12 +6,98 @@ Contributing to Sherpa development
 
    Needs work.
    
-Contributions to Sherpa - whether it be bug reports,
-documentation updates, or new code - are highly encouraged.
+Contributions to Sherpa - whether it be bug reports, documentation
+updates, or new code - are highly encouraged.  Please `report any
+problems or feature requests on github
+<https://github.com/sherpa/sherpa/issues/>`_.
+
 
 At present we do not have any explicit documentation on how
 to contribute to Sherpa, but it is similar to other open-source
 packages such as
 `AstroPy <http://docs.astropy.org/en/stable/index.html#contributing>`_.
 
 The developer documentation is also currently lacking.
+
+To do code development, Sherpa needs to be installed from source so
+that tests can run locally and the documentation can be build locally
+to test out any additions to code or docs.  :ref:`build-from-source`
+describes several ways to build Sherpa from source, but one
+particularly comfortable way is described in detail in the next
+section.
+
+.. _source-install-with-conda:
+
+Install from source in conda
+============================
+
+Conda can be used to install all the dependencies for Sherpa.
+ 
+::
+
+    conda create -n sherpaciao -c https://cxc.cfa.harvard.edu/conda/ciao ciao ds9 ciao-contrib caldb_main astropy
```

Comment:
For Sherpa development, we don't need to install `ciao-contrib` or `caldb_main` (dropping the latter would save users a lot of bandwidth/time). Similarly, can they get away with just `sherpa` and `ds9` rather than `ciao` (I can't remember what `sherpa` pulls in, so maybe it isn't enough?). I guess it depends if you want to use crates or astropy backends.

Reply:
On second thought, I don't think I need to explain how to install ciao into that environment, too. These are Sherpa docs. Developers who are also CIAO conda users will know, and others don't need to know. Pushed changes.